### PR TITLE
FIX Bug keeping data on GPU (3D)

### PR DESCRIPTION
**Pull request by janden**

### CODE REVIEW

**Comment by janden** [comment]
This now runs with the following output:
```
Ridge regression, alpha: 1.0, MAE: 5.928058624267578, RMSE: 8.224647521972656
/mnt/xfs1/home/janden/local/anaconda3/envs/kymatio_cuda90/lib/python3.7/site-packages/sklearn/linear_model/ridge.py:125: LinAlgWarning: Ill-conditioned matrix (rcond=1.7272e-08): result may not be accurate.
  overwrite_a=True).T
/mnt/xfs1/home/janden/local/anaconda3/envs/kymatio_cuda90/lib/python3.7/site-packages/sklearn/linear_model/ridge.py:125: LinAlgWarning: Ill-conditioned matrix (rcond=1.57417e-08): result may not be accurate.
  overwrite_a=True).T
/mnt/xfs1/home/janden/local/anaconda3/envs/kymatio_cuda90/lib/python3.7/site-packages/sklearn/linear_model/ridge.py:125: LinAlgWarning: Ill-conditioned matrix (rcond=1.8823e-08): result may not be accurate.
  overwrite_a=True).T
/mnt/xfs1/home/janden/local/anaconda3/envs/kymatio_cuda90/lib/python3.7/site-packages/sklearn/linear_model/ridge.py:125: LinAlgWarning: Ill-conditioned matrix (rcond=1.36575e-08): result may not be accurate.
  overwrite_a=True).T
/mnt/xfs1/home/janden/local/anaconda3/envs/kymatio_cuda90/lib/python3.7/site-packages/sklearn/linear_model/ridge.py:125: LinAlgWarning: Ill-conditioned matrix (rcond=1.51197e-08): result may not be accurate.
  overwrite_a=True).T
Ridge regression, alpha: 0.1, MAE: 4.962062835693359, RMSE: 6.941751003265381
Ridge regression, alpha: 0.01, MAE: 4.063272476196289, RMSE: 5.800691604614258
Ridge regression, alpha: 0.001, MAE: 3.3604490756988525, RMSE: 4.858139514923096
Ridge regression, alpha: 0.0001, MAE: 2.9941437244415283, RMSE: 4.416964530944824
Ridge regression, alpha: 1e-05, MAE: 2.8302745819091797, RMSE: 4.209273338317871
Ridge regression, alpha: 1e-06, MAE: 2.7701542377471924, RMSE: 4.150631904602051
Ridge regression, alpha: 1e-07, MAE: 2.7910399436950684, RMSE: 4.229248523712158
Ridge regression, alpha: 1e-08, MAE: 3.0019569396972656, RMSE: 4.44858455657959
Ridge regression, alpha: 1e-09, MAE: 3.3194243907928467, RMSE: 4.7733330726623535
```

Didn't we decide to remove `alpha = 1.0` from the cross validation?

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ jobs:
     branches:
       only:
         - master
+        - dev
```
Comment (edouardoyallon) [inline .circleci/config.yml]: ?! should it be in this PR?

File: .circleci/config.yml [hunk 2]
```diff
@@ -75,4 +76,12 @@ jobs:
             - "37:ad:c6:65:3e:d1:b4:7a:0a:8a:9d:df:a8:36:91:6d"
 
       - run:
-          bash tools/push_doc.sh doc/build/html
+          name: push to the right place
```
Comment (edouardoyallon) [inline .circleci/config.yml]: and this as well?

File: examples/3d/scattering3d_qm7_torch.py [hunk 3]
```diff
@@ -239,8 +239,8 @@
 order_0 = torch.cat(order_0, dim=0)
 orders_1_and_2 = torch.cat(orders_1_and_2, dim=0)
 
-order_0 = order_0.numpy()
-orders_1_and_2 = orders_1_and_2.numpy()
+order_0 = order_0.cpu().numpy()
+orders_1_and_2 = orders_1_and_2.cpu().numpy()
```
Comment (edouardoyallon) [inline examples/3d/scattering3d_qm7_torch.py]: ok

File: examples/3d/scattering3d_qm7_torch.py [hunk 4]
```diff
@@ -303,7 +303,7 @@
 # Tikhonov, regularization). The mean absolute error (MAE) and root mean
 # square error (RMSE) is output for each value of `alpha`.
 
-alphas = 10.0 ** (-np.arange(0, 10))
+alphas = 10.0 ** (-np.arange(1, 10))
```
Comment (edouardoyallon) [inline examples/3d/scattering3d_qm7_torch.py]: ok

File: .circleci/config.yml (hunk 1 above)
Comment (janden) [reply .circleci/config.yml]: Because of @eickenberg's shenanigans. Will make sure it merges on top.

File: .circleci/config.yml (hunk 2 above)
Comment (janden) [reply .circleci/config.yml]: Same.

**Comment by janden** [comment]
Will fix up the history here and then it should be ready for merge.

**Pull request merged**